postman/threads-api.postman_collection.json

```diff
@@ -2218,6 +2519,269 @@
+										"{{api_host}}"
+									],
+									"path": [
+										"me",
+										"mentions"
+									],
+									"query": [
+										{
+											"key": "since",
+											"value": "",
+											"description": "Query string parameter representing the start date for retrieval.",
+											"disabled": true
+										},
+										{
+											"key": "until",
+											"value": "",
+											"description": "Query string parameter representing the end date for retrieval.",
+											"disabled": true
+										},
+										{
+											"key": "limit",
+											"value": "",
+											"description": "Query string parameter representing the maximum number of media objects or records to return.",
+											"disabled": true
+										},
+										{
+											"key": "fields",
+											"value": "{{fields_threads}}",
+											"description": "A comma-separated list of fields for media objects on Threads."
+										}
+									]
+								}
+							},
+							"_postman_previewlanguage": null,
+							"header": null,
+							"cookie": [],
+							"body": "{\n    \"data\": [\n        {\n            \"id\": <string>,\n            \"media_product_type\": \"THREADS\",\n            \"media_type\": \"TEXT_POST\",\n            \"permalink\": <string>,\n            \"username\": <string>,\n            \"text\": <string>,\n            \"timestamp\": \"2025-01-01T00:00:00+0000\",\n            \"shortcode\": <string>,\n            \"is_quote_post\": false,\n            \"has_replies\": false\n        },\n        ...\n    ],\n    \"paging\": {\n        \"cursors\": {\n            \"before\": <string>,\n            \"after\": <string>\n        }\n    }\n}"
+						}
+					]
+				}
+			]
+		},
+		{
+			"name": "Display Threads",
+			"item": [
+				{
+					"name": "Embed a Threads Post",
+					"request": {
+						"auth": {
+							"type": "noauth"
+						},
+						"method": "GET",
+						"header": [],
+						"url": {
+							"raw": "{{api_host}}/oembed?url=https://www.threads.net/@threads/post/DCkkKl_OGb1",
+							"host": [
+								"{{api_host}}"
+							],
+							"path": [
+								"oembed"
+							],
+							"query": [
+								{
+									"key": "url",
+									"value": "https://www.threads.net/@threads/post/DCkkKl_OGb1",
+									"description": "This is the URL of the Threads post to be embedded. With standard access, you may embed posts from the @meta, @threads, @instagram, and @facebook accounts."
+								},
+								{
+									"key": "access_token",
+									"value": "TH|{{app_id}}|{{app_secret}}",
+									"description": "This is your app access token. This consists of your app's ID and secret.",
+									"disabled": true
+								},
+								{
+									"key": "access_token",
+									"value": "{{app_access_token}}",
+									"description": "This is your app access token received via the GET /oauth/access_token endpoint. See the Authorization folder for details.",
+									"disabled": true
+								}
```

Comment:
It looks like there are two keys for `access_token`. Let's remove this one and keep the other one.

<img width="1191" alt="image" src="https://github.com/user-attachments/assets/d5d4f5dd-1085-4640-ab6f-8e7ba706c378">

Reply:
These are different. App access tokens can be either the combination of the id and secret (top) or an encrypted token (bottom; via the `GET /oauth/access_token` endpoint)

Reply:
Ah ok got it. So does that mean you can use either one for oEmbed?

Reply:
Yup either one works